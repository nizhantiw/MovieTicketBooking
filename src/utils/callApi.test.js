import callApi from './callApi';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer'
configure({adapter: new Adapter()});

it("renders correctly", () => {
  const wrapper = shallow(
    <callApi />
  );
  expect(wrapper).toMatchSnapshot();
});

it('renders a snapshot', () => {
    const tree = renderer.create(<callApi/>).toJSON();
    expect(tree).toMatchSnapshot();
});
  
it('renders the component', () => {
  const wrapper = shallow(<callApi />);
  expect(wrapper.exists()).toBe(true);
});
