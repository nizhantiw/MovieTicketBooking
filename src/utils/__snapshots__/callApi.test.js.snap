// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `<callApi />`;

exports[`renders correctly 1`] = `ShallowWrapper {}`;
